addon/lib/experiment-hacks.js

```diff
@@ -8,19 +8,20 @@ const PrefsService = require('sdk/preferences/service');
 const store = require('sdk/simple-storage').storage;
 
 const BLOK_ID = 'blok@mozilla.org';
-const TP_PREFS = [
-  'privacy.trackingprotection.enabled',
-  'privacy.trackingprotection.pbmode.enabled',
-  'services.sync.prefs.sync.privacy.trackingprotection.enabled',
-  'services.sync.prefs.sync.privacy.trackingprotection.pbmode.enabled'
-];
+const TP_PREFS = {
+  'privacy.trackingprotection.enabled': false,
+  'privacy.trackingprotection.pbmode.enabled': true,
+  'services.sync.prefs.sync.privacy.trackingprotection.enabled': false,
+  'services.sync.prefs.sync.privacy.trackingprotection.pbmode.enabled': false
```

Comment:
Ah, so this `false` is saying not to sync. I thought it was setting the pref "in the cloud" to be `false`.